lib/kaffe/group_member/subscriber/group_member.ex

```diff
@@ -0,0 +1,152 @@
+defmodule Kaffe.GroupMember do
+  @moduledoc """
+  Consume messages from a Kafka topic for a consumer group.
+
+  The actual consumption is delecated to a series of subscribers, see
+  `Kaffe.Subscriber`.
+
+  The subscribers are assigned generations. Each generation represents a
+  specific configuration. In order to allow the partitions to be rebalanced on
+  startup, there is a delay between receiving a set of assignments associated
+  with that generation and actually creating the subscribers. If a new
+  generation is received in the mean time, the older generation is discarded.
+
+  See: https://github.com/klarna/brod/blob/master/src/brod_group_member.erl
+
+  Also: https://github.com/klarna/brucke/blob/master/src/brucke_member.erl
+
+  The `brod_group_member` behavior is used. 
+  """
+
+  use GenServer
+
+  @behaviour :brod_group_member
+
+  alias Kaffe.WorkerManager
+
+  require Logger
+
+  defmodule State do
+    defstruct subscribers: [],
+      subscriber_name: nil,
+      group_coordinator_pid: nil,
+      consumer_group: nil,
+      worker_manager_pid: nil,
+      topic: nil,
+      configured_offset: nil,
+      current_gen_id: nil
+  end
+
+  def start_link(subscriber_name, consumer_group, worker_manager_pid, topic, configured_offset) do
+    GenServer.start_link(__MODULE__, [subscriber_name, consumer_group,
+      worker_manager_pid, topic, configured_offset])
+  end
+
+  def init([subscriber_name, consumer_group, worker_manager_pid, topic, configured_offset]) do
+
+    :ok = kafka().start_consumer(subscriber_name, topic, [])
+    {:ok, pid} = group_coordinator().start_link(subscriber_name, consumer_group,
+      [topic], _group_config = [], __MODULE__, self())
+
+    Logger.info "event#init group_coordinator=#{inspect pid} subscriber_name=#{subscriber_name} consumer_group=#{consumer_group}"
+
+    {:ok, %State{subscriber_name: subscriber_name,
+                group_coordinator_pid: pid,
+                consumer_group: consumer_group,
+                worker_manager_pid: worker_manager_pid,
+                topic: topic,
+                configured_offset: configured_offset}}
+  end
+
+  def get_committed_offsets(_group_member_pid, _topic_partitions) do
+    # Should not receive this
```

Comment:
Good info! That'd be perfect to put into the comment as why we shouldn't receive that message.